doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -566,78 +566,104 @@ The query returns all the entities that match all the matching conditions (AND l
 
 There are two kinds of statements: *unary statements* and *binary statements*.
 
-Binary statements are composed by an attribute name (e.g. "temperature"), an operator and a value
+Binary statements are composed by an attribute path (e.g. `temperature`), an operator and a value
```

Comment:
perhaps for the sake of a better illustration use something like `brand.name` as an example, to note the structure of a path, or even use both examples `temperature` or `brand.name`